<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QSlider" name="intraocularPressureSlider">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>470</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="minimum">
    <number>10</number>
   </property>
   <property name="maximum">
    <number>50</number>
   </property>
   <property name="value">
    <number>20</number>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="loadButton">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>120</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Select Image</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>170</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="resetButton">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>170</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
  <widget class="QPushButton" name="simulateButton">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>120</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Simulate</string>
   </property>
  </widget>
  <widget class="QLabel" name="processedLabel">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>250</y>
     <width>281</width>
     <height>161</height>
    </rect>
   </property>
   <property name="text">
    <string>Processed Image</string>
   </property>
  </widget>
  <widget class="QLabel" name="originalLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>250</y>
     <width>271</width>
     <height>161</height>
    </rect>
   </property>
   <property name="text">
    <string>Original Image</string>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dateTimeEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>194</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableTimeline">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>450</y>
     <width>511</width>
     <height>121</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
   </property>
   <column>
    <property name="text">
     <string>Date:</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>New Column</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>IOP(mmHg):</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Note:</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="labelRiskLevel">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>530</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Risk Level</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>451</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>Patient Information</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <widget class="QTextEdit" name="textEditNotes">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>84</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditSurname">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditName">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditID">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>110</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEditBirth">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>30</y>
      <width>110</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Surname:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>ID:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>60</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Note:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>30</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Birth:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxGender">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>150</y>
      <width>76</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Woman</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Man</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Unknown</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Gender:</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="showGraphButton">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>290</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Show Graph</string>
   </property>
  </widget>
  <widget class="QPushButton" name="savePatientButton">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>70</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Patient</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loadPatientButton">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>70</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Load Patient</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxCSF">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>690</y>
     <width>42</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="opticNerveLabel">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>600</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Optic Nerve Simulation</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnShowTPD">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>650</y>
     <width>121</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Show TPD Graph:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exportButton">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>600</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Save All Data</string>
   </property>
  </widget>
  <widget class="QPushButton" name="importButton">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>660</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Load Data</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>40</y>
     <width>62</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelTPD">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>730</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TPD</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
